deployment/chainloop/README.md

```diff
@@ -89,6 +89,19 @@ helm install [RELEASE_NAME] oci://ghcr.io/chainloop-dev/charts/chainloop \
     # ...
 ```
 
+Deploy using GCP secret manager instead of Vault
+
+```console
+helm install [RELEASE_NAME] oci://ghcr.io/chainloop-dev/charts/chainloop \
+    # Open ID Connect (OIDC)
+    # ...
+    # Secrets backend
```

Comment:
This example in-fact is missing an additional flag
```suggestion
    # Secrets backend
    --set secretsBackend.backend=gcpSecretManager \
```

in fact this is a problem in the aws example too, I just recorded it in this issue #226

Reply:
Good point, thanks a lot for noticing that.